easy-ui-react/src/NexusLayout/NexusLayout.tsx

```diff
@@ -0,0 +1,208 @@
+import {
+  NexusLayoutMultipageContainer,
+  NexusLayoutMultipageContent,
+  NexusLayoutMultipageHeader,
+  NexusLayoutMultipageSidebar,
+  NexusLayoutMultipageTitle,
+} from "./NexusLayoutMultipage";
+import {
+  NexusLayoutMultipageSidebarNav,
+  NexusLayoutMultipageSidebarNavLink,
+  NexusLayoutMultipageSidebarNavSection,
+  useNexusLayoutMultipageSidebarNav,
+} from "./NexusLayoutMultipageSidebarNav";
+import {
+  NexusLayoutNav,
+  NexusLayoutNavLink,
+  useNexusLayoutNav,
+} from "./NexusLayoutNav";
+
+import styles from "./NexusLayout.module.scss";
+
+export type NexusLayoutProps = {
+  /** Layout children. */
+  children: ReactNode;
+
+  /**
+   * Background decoration for layout.
+   *
+   * @default 01
+   */
+  backgroundDecoration?: "01";
+};
+
+export type NexusLayoutHeaderProps = {
+  /**
+   * Custom logo for layout. Defaults to EasyPost logo.
+   */
+  logo?: ReactNode;
+
+  /** Header children. */
+  children: ReactNode;
+};
+
+export type NexusLayoutContentProps = {
+  /** Content children. */
+  children: ReactNode;
+};
+
+/**
+ * `NexusLayout` defines the header, main content, and multi-page content areas of a Nexus product page.
+ *
+ * @example
+ * ```tsx
+ * <NexusLayout>
+ *   <NexusLayout.Header>
+ *     <NexusLayout.Nav selectedKey="/1">
+ *       <NexusLayout.NavLink key="/1" href="/1" iconSymbol={Icon}>
+ *         Order Fulfillment
+ *       </NexusLayout.NavLink>
+ *       <NexusLayout.NavLink key="/2" href="/2" iconSymbol={Icon}>
+ *         Post Delivery
+ *       </NexusLayout.NavLink>
+ *     </NexusLayout.Nav>
+ *     <NexusLayout.Actions>
+ *       <NexusLayout.MenuAction
+ *         accessibilityLabel="Label"
+ *         iconSymbol={Icon}
+ *         renderBadge={() => <Badge />}
+ *       >
+ *         <Menu.Overlay onAction={() => {}}>
+ *           <Menu.Item>Action 1</Menu.Item>
+ *           <Menu.Item>Action 2</Menu.Item>
+ *         </Menu.Overlay>
+ *       </NexusLayout.MenuAction>
+ *       <NexusLayout.MenuAction accessibilityLabel="Label" iconSymbol={Icon}>
+ *         <Menu.Overlay onAction={() => {}}>
+ *           <Menu.Item>Action 1</Menu.Item>
+ *           <Menu.Item>Action 2</Menu.Item>
+ *         </Menu.Overlay>
+ *       </NexusLayout.MenuAction>
+ *     </NexusLayout.Actions>
+ *   </NexusLayout.Header>
+ *   <NexusLayout.Content>
+ *     <div>Content</div>
+ *   </NexusLayout.Content>
+ * </NexusLayout>
+ * ```
+ */
+export function NexusLayout(props: NexusLayoutProps) {
+  const { backgroundDecoration = "01", children } = props;
+  const className = classNames(
+    styles.NexusLayout,
+    styles[variationName("backgroundDecoration", backgroundDecoration)],
+  );
+  return <div className={className}>{children}</div>;
+}
+
+function NexusLayoutHeader(props: NexusLayoutHeaderProps) {
+  const { logo = <EasyPostLogo />, children } = props;
+  return (
+    <header className={styles.header}>
+      <HorizontalStack blockAlign="center" align="space-between">
```

Comment:
Question: `children` have both `nav` and `actions` in it, and with `space-between` on `HorizontalStack` will the `nav` be centered on the page if `logo` and `actions` have different widths?

Reply:
good call. addressed